doc/sphinx/source/LwM2M.rst

```diff
@@ -381,7 +381,7 @@ requests that may be sent from the Client to the Server include:
 - **Update**, which is a CoAP **POST** request on an URL previously returned in
   a response to *Register*, is sent periodically to ensure the Server that the
   device is still online, and whenever any of the information previously given
-  in a Register message change - so that the Server always has up-to-date
+  in a Register message changes - so that the Server always has up-to-date
```

Comment:
...or actually it is, I derped.
But since we have such doubts, it probably needs more thorough simplification...